Added test notification option (#7629)
mattermost/docs: source/preferences/troubleshoot-notifications.rst

The Mattermost notifications you receive depend on your `Mattermost preferences <#check-your-mattermost-preferences>`__, the `Mattermost client <#check-your-mattermost-client-settings>`__ you're using, and the `operating system (OS) <#check-your-operating-system-settings>`__ you're running Mattermost on.

Send yourself a test notification
----------------------------------

> From Mattermost v10.3, you can send yourself a test notification by selecting **Settings** |gear| in the top right corner of Mattermost.

Author: ```suggestion
From Mattermost v10.3, you can send yourself a test notification by selecting **Settings** |gear|, and going to the **Notifications** options.
```

> Under **Troubleshooting notifications** select the **Send a test notification** option. If notifications are working, you'll receive a system-bot direct message to confirm. 

Author: ```suggestion
Under **Troubleshooting notifications** select the **Send a test notification** option. If notifications are working, you'll receive a system-bot direct message in Mattermost confirming that notifications are working. 
```
